content/docs/ui/sending-email/getting-started-with-automation.md

```diff
@@ -190,6 +189,57 @@ At this phase of the beta, custom fields are not supported. You can import [rese
 4. Upload your file by dragging it into or clicking the CSV upload area.
    Once the CSV has processed, you will receive a [notification]({{root_url}}/ui/sending-email/getting-started-with-automation/#notifications) email.
 
+### Create a signup form
+
+Use Signup Forms with the SendGrid Automation beta to capture new contacts and add them directly to an Automation list. If the list you add them to is acting as the entry criteria for a live automation, the new contact will automatically receive any emails you’ve created as part of that automation. 
+
+You can enable contacts to access your signup form either by sending them a link hosted by SendGrid or by embedding it directly into your website.
+
+#### Create a new signup form
+
+<call-out>
+
+The number of signup forms you can create is limited to 15. 
+
+</call-out>
+*To create a new signup form:*
+
+1. Navigate to Automations and select **Signup Forms**. 
+1. Click **Create New Form**.
+1. On the Settings tab, add a Form Name and then select the contact list you want to add new signups to. 
+1. Next, add a confirmation message. This field contains the text that contacts will see once they’ve clicked submit on the form.
+1. Navigate to the Build tab and add a header to the form. 
+1. Add the intro copy to explain what the contact is signing up to receive. 
+1. Select the checkboxes to add the fields you want on the form.
+
+#### Manage signup forms
+
+Using the action menu on the Signup Forms page you can edit, duplicate, or delete existing signup forms. 
+
+![]({{root_url}}/img/action-menu-signup-form.png "Action menu for signup forms")
+
+<call-out>
+
+Because of the iframe technology used to create the signup form, anytime you edit a form and click **Save & Apply** the form will be dynamically updated on whatever page or pages you have the form embedded in. 
```

Comment:
this is passive and has a preposition at the end of the sentence. How about this?

Because of the iframe technology used to create the signup form, the form dynamically updates on the page or pages where you have the form embedded anytime you edit a form and click **Save & Apply**.